### ISSUE TITLE
Feature request / docs request - Usage with Node.js streams

### ISSUE THREAD

**Issue by ORESoftware** [Turistforeningen/node-s3-uploader#130]
### Subject of the issue
Describe your issue here.

### Your environment
* Operating system
  * [ ] Mac
  * [ ] Windows
  * [x] Linux
* Node.js version (`node --version`)
  * [ ] Node.js v4.x
  * [x] Node.js v6.x
* ImageMagick version (`identify -version`)
  * [ ] ImageMagick v6.9.x
  * [x] ImageMagick v7.0.x
* s3-uploader version
  * [ ] v0.x
  * [ ] v1.x
  * [x] v2.x

Just looking for a way to use this library with Node.js streams, is it possible?

**Comment by Starefossen** [comment]
No, unfortunately not. This package executes the installed `ImageMagic` binary (similar to the terminal command), it does not bind directly to the ImageMagic library which would allow us to use Node.js streams.

**Comment by ORESoftware** [comment]
got it np thanks for responding!
